docs/modules/ROOT/pages/references/architecture/upgrade_controller.adoc

```diff
@@ -136,15 +136,41 @@ If any of the operators is degraded, the cluster should be considered unhealthy
 
 It should be possible to exclude operators.
 
+=== The controller can pause machine configuration pools to delay node reboots [[mcp-upgrade-pause]]
```

Comment:
```suggestion
=== The controller can pause and unpause machine configuration pools to delay node reboots [[mcp-upgrade-pause]]
```
```suggestion
=== The controller can temporarily pause machine configuration pools to delay node reboots [[mcp-upgrade-pause]]
```
With the current description it's not fully clear that the upgrade controller handles both pausing and unpausing IMO.